src/wallet/test/coinselector_tests.cpp

```diff
@@ -658,4 +663,47 @@ BOOST_AUTO_TEST_CASE(SelectCoins_test)
     }
 }
 
+BOOST_AUTO_TEST_CASE(waste_test)
+{
+    CoinSet selection;
+    const CAmount fee = CAmount(100);
+    const CAmount change_cost = CAmount(100);
```

Comment:
You may want to use different values (preferably not direct multiples) for `fee` and `change_cost` as otherwise some weird interaction might hide a defect, maybe something like 100 and 125.

Reply:
Done